bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/system/ITarget.java

```diff
@@ -28,7 +26,7 @@
  */
 public abstract class ITarget implements Cloneable, Serializable {
```

Comment:
It has been my impression that we want to get away from the old Target model rather than extend it. @swalker2m do you have an opinion?